src/main/java/frc/robot/Config.java

```diff
@@ -105,6 +105,34 @@ else if (RobotBase.isSimulation()) {
 
   public static final Boolean swerveTuning = true;
 
+  public static final class PhotonConfig{
+    public static double[] APRIL_HEIGHTS = {0,0,0,0,1.32,1.32,1.32,0,0,0,0,1.31,1.31,1.31,1.31,1.31,1.31};
+    public static enum PhotonPositions {
+      
+      LEFT_SPEAKER_RED(4, new Translation2d(-0.937,-0.937), new Translation2d(-0.637,-0.637), Rotation2d.fromDegrees(45)),
```

Comment:
Speaker positions would probably be much faster if they didn't have a waypoint. My recommendation on how to go about that is to add a `public final boolean hasWaypoint;` to PhotonPositions, then you can write a second constructor, something like this:
```Java
      PhotonPositions(int id, Translation2d destination, Rotation2d direction) {
        this.id = id;
        this.destination = destination;
        this.direction = direction;

        this.hasWaypoint = false;
      }
```

Can then change getAprilTagCommand to check the boolean and either have 2 PhotonMoveToTarget or just one.